cpp/include/cudf/io_types.h

```diff
@@ -218,33 +219,20 @@ typedef struct {
 
 } pq_read_arg;
 
-/**---------------------------------------------------------------------------*
- * @brief Input and output arguments to the read_json interface.
+namespace cudf{
+ /**---------------------------------------------------------------------------*
+ * @brief Arguments to the read_json interface.
  *---------------------------------------------------------------------------**/
-typedef struct {
-  /*
-   * Output arguments
-   */
-  int           num_cols_out;               ///< Out: Number of columns returned
-  int           num_rows_out;               ///< Out: Number of rows returned
-  gdf_column    **data;                     ///< Out: Array of gdf_column*
-  int           *index_col;                 ///< Out: If available, column index to use as row labels
-
-  /*
-   * Input arguments
-   */
+struct json_read_arg{
```

Comment:
Now that this is a C++ struct, does it make sense to add constructors and maybe default args to make it easier to configure?

Reply:
I think it does. It's not as needed as with the CSV args, but it would still be beneficial.
I'll add some type of initialization, thanks.